sys/riotboot/flashwrite.c

```diff
@@ -155,8 +155,6 @@ int riotboot_flashwrite_finish_raw(riotboot_flashwrite_t *state,
 {
     assert(len <= FLASHPAGE_SIZE);
 
-    int res = -1;
-
     uint8_t *slot_start = (uint8_t *)riotboot_slot_get_hdr(state->target_slot);
 
 #if CONFIG_RIOTBOOT_FLASHWRITE_RAW
```

Comment:
Hmm looking at this if `CONFIG_RIOTBOOT_FLASHWRITE_RAW` is set there is no verification that the content was written correctly, shouldn't we rename `flashpage_write_and_verify` a `flashpage_write_and_verify_page` and then add `flashpage_write_and_verify` to write and verify? Maybe you planned for this in a later stage @bergzand ?

Reply:
Here its not that big of a deal but above it will allow failing early if a write failed.

Reply:
> Hmm looking at this if `CONFIG_RIOTBOOT_FLASHWRITE_RAW` is set there is no verification that the content was written correctly, shouldn't we rename `flashpage_write_and_verify` to `flashpage_write_and_verify_page` and then add `flashpage_write_and_verify` to write and verify? Maybe you planned for this in a later stage @bergzand ?

Not my intention. I've never seen a full firmware write fail on the flash itself. There is always the later digest check in the  update process, but that doesn't allow for early failures.

Reply:
`flashpage_write()` is also a `void` function, so no error will be reported.